torch/nn/functional.py

```diff
@@ -41,6 +41,9 @@
 
 Note:
     {cudnn_reproducibility_note}
+
+Note:
+    This operator supports complex datatypes.
```

Comment:
Are there multiple ways the complex support could be implemented? e.g. "true complex" support (whatever that means) vs. normal convolutions done separately on the real & imaginary parts. If so, I think it's useful to indicate which one is done in practice so users know what to expect.

Reply:
maybe just list all the complex data types explicitly though (complex32, complex64, complex128) just because documentation gets outdated pretty quickly (you could imagine having more complex data types in the future)